build_scripts/build_windows.ps1

```diff
@@ -26,12 +26,9 @@ else
 Write-Output "   ---"
 Write-Output "Create venv - python3.9 is required in PATH"
 Write-Output "   ---"
-python -m venv venv
+.\Install.ps1 -d
```

Comment:
In the interest in keeping this more similar to the build scripts for the other platforms, should we move this to the workflow file? (I think that is where install.sh is called for the other platforms)

Reply:
I don't super care where that happens.  It is already present over in https://github.com/Chia-Network/chia-blockchain/pull/11077 so it will happen eventually.

Reply:
Can wait for the other changes then, this one looks good for now